.gitignore

```diff
@@ -33,3 +33,5 @@ yarn-error.log
 core
 
 .idea
+
+auth.json
```

Comment:
Every environment has to add an `auth.json` entry to authenticate the backpack/pro package.